source/content/guides/autopilot/03-enable-autopilot.md

```diff
@@ -82,6 +82,8 @@ The Autopilot setup wizard automatically displays after you click **Customize**
     - Weekly
```

Comment:
Let's flip the order of the Frequency settings (in all places) and add "Daily" and "Custom" with asterisks or a note that explains they are only available to sites owned by Platinum and Diamond Workspaces. 

```suggestion
    - Monthly
    - Weekly
    - Daily*
    - Custom*
```

Reply:
@kelly-mcgonnigle - Confirming that the 'Never' option should stay the same? Please advise.

Reply:
Yes, `Never` should still be there in the list! Not sure where it went..